packages/transaction-controller/src/types.ts

```diff
@@ -1368,3 +1371,14 @@ export type SubmitHistoryEntry = {
   /** The transaction parameters that were submitted. */
   transaction: TransactionParams;
 };
+
+export enum BlockaidResultType {
```

Comment:
Rather than defining a full enum, could we just define a local constant such as `BLOCKAID_RESULT_TYPE_MALICIOUS`?

Reply:
Done